datastore/src/main/kotlin/com/openlattice/datastore/data/controllers/DataIntegrationController.kt

```diff
@@ -47,14 +46,21 @@ class DataIntegrationController : DataIntegrationApi, AuthorizingComponent {
         return authz
     }
 
-    override fun generatePresignedUrls(data: Collection<S3EntityData?>?): List<String?>? {
+    override fun generatePresignedUrls(
+            data: Collection<S3EntityData>,
+            propertyUpdateType: PropertyUpdateType
+    ): List<String> {
         throw UnsupportedOperationException("This shouldn't be invoked. Just here for the interface and efficiency")
     }
 
     @Timed
     @PostMapping("/" + DataIntegrationApi.S3)
     fun generatePresignedUrls(
-        @RequestBody data: List<S3EntityData>
+            @RequestBody data: List<S3EntityData>,
+            @RequestParam(
+                    name = PROPERTY_UPDATE_TYPE,
+                    defaultValue = "Versioned"
```

Comment:
`PropertyUpdateType.Versioned` rather than hard-coded string

Reply:
I don't think that's possible.

![Screen Shot 2021-09-29 at 5 54 53 PM](https://user-images.githubusercontent.com/440299/135368213-48e5f977-f09a-49be-aa9d-8218388ee2d3.png)